### PR TITLE
Adding dockerfile for future DockerHub publishing

### PR DESCRIPTION
**Pull request by jesseadams**
Pre-requisite of #162

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM ruby:2.5
+
+MAINTAINER eric.kascic@stelligent.com
+
+RUN gem install cfn-nag
```
Comment (erickascic) [inline Dockerfile]: i'd suggest we pin this based upon the version the pipeline pushes to rubygems.... and then the label of the image can reflect that version

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+ENTRYPOINT ["cfn_nag"]
```
Comment (erickascic) [inline Dockerfile]: maybe we need one for scan v. regular?

Comment (jesseadams) [reply Dockerfile]: Not sure I follow

File: Dockerfile (hunk 1 above)
Comment (jesseadams) [reply Dockerfile]: I was going to do this when we figured out DockerHub and start publishing. This was just a starting point.

File: Dockerfile (hunk 2 above)
Comment (erickascic) [reply Dockerfile]: there are two command lines: cfn_nag and cfn_nag_scan.  the latter will search recursively through a directory structure for cfn templates and aggregate the results

Comment (jesseadams) [reply Dockerfile]: We should be able to use both with the same Dockerfile, just default to one or the other, no?

**Pull request merged**